<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>共修记录器  顶礼上师三宝</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app.qrc">
    <normaloff>:/ico/images/default.ico</normaloff>:/ico/images/default.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <widget class="QPushButton" name="btnExec">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>开始录音</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOutput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAboutUs">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关于</string>
     </property>
    </widget>
    <widget class="QLabel" name="labTime">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;simsun&quot;, &quot;Microsoft YaHei&quot;, &quot;宋体&quot;;
font-size: 20px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>时间轴：00:00:00</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tabShowTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>133</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbSet">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>451</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QLineEdit" name="txtSavePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSavePath">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>保存目录</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radQ">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>54</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>高音质</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labSoundQuality">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>54</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>音质：</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radL">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>54</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>高压版</string>
     </property>
    </widget>
    <widget class="QLabel" name="labinfo">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>78</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>高音质指原声波录制，低音质是有损压缩，比高音质低五倍！</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbkSynchronous">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>54</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>录音同步记录</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnrtable">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>426</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>起始时间</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
